Task12.py

```diff
@@ -0,0 +1,27 @@
+"""
+      Task 12 - Recursion.
+
+      Description:
+
+Given list of list of list etc of integers
+write recursive function that will accept as argument target list
+and return sum of all integers inside it
+Input: [[[[1, 4, 5], [[6, 9],[[[8, 1], 7], 3], 2], 7], 5, 2], 9, [1, 2]]
+Output: Target sum = 72
+"""
+
+l1 = [[[[1, 4, 5], [[6, 9], [[[8, 1], 7], 3], 2], 7], 5, 2], 9, [1, 2]]
+
+
+def get_target_sum(obj, target_sum=None):
```

Comment:
Correct solution, but a little bit complicated.
It is possible to do the same but without function additional argument ( target_sum)
Please examine code of possible alternative 
```
def complex_list_sum(list_):
    curr_sum = 0
    for el in list_:
        curr_sum += complex_list_sum(el) if type(el) is list else el
    return curr_sum
```